images/deployer/Dockerfile

```diff
@@ -9,4 +9,11 @@
 #
 FROM openshift/origin
 
+RUN yum install -y nfs-utils sudo
+
+# Edit sudoers file
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
```

Comment:
I thought the same thing.  The argument in favor of putting it in deployer in the first place is that it is guaranteed to be on all nodes.  This prevents downloading a 2nd image.

I think the better solution, too, is to have configurable recyclers and offload as much of this as we can to the end user.  They can craft recyclers tailored to their environment.